frontend/website/src/Links.re

```diff
@@ -46,6 +46,11 @@ module Static = {
     link: Cdn.url("/static/coda-whitepaper-05-10-2018-0.pdf"),
   };
 
+  let modifiedsnark = () => {
+    name: ("Read About the zkSNARK construction We're Using", "modified-BG-snark"),
```

Comment:
We should make sure we don't over/under capitalize here "Read about the zkSNARK construction we're using"

Reply:
Done.